package.json

```diff
@@ -63,6 +63,7 @@
     "del": "^6.0.0",
     "elf-cam": "^0.1.1",
     "end-of-stream": "^1.4.4",
+    "es-module-lexer": "^0.7.1",
```

Comment:
the node docs reference https://github.com/guybedford/cjs-module-lexer, which led me to this (https://github.com/guybedford/es-module-lexer)